vulnerable_people_form/form_pages/check_your_answers.py

```diff
@@ -30,10 +30,9 @@ def get_check_your_answers():
 
 @form.route("/check-your-answers", methods=["POST"])
 def post_check_your_answers():
-    registration_number = persist_answers_from_session()
+    registration_number, is_spl_match = persist_answers_from_session()
```

Comment:
is_spl_match can be removed as it doesn't exist anymore

Reply:
Done